tests/end-to-end/logging/tap.phpt

```diff
@@ -0,0 +1,109 @@
+--TEST--
+TAP
+--FILE--
+<?php declare(strict_types=1);
+$_SERVER['argv'][] = '--do-not-cache-result';
+$_SERVER['argv'][] = '--no-configuration';
+$_SERVER['argv'][] = '--tap';
+$_SERVER['argv'][] = __DIR__ . '/../_files/OutcomesAndIssuesTest.php';
+
+require_once __DIR__ . '/../../bootstrap.php';
+
+(new PHPUnit\TextUI\Application)->run($_SERVER['argv']);
+--EXPECTF--
```

Comment:
FWIW: PHPT tests themselves are intended to be valid PHP files, so I would recommend adding the closing PHP tag.

Reply:
I know that https://qa.php.net/write-test.php has

> The FILE section is used as the body of the .php file, so don't forget to open and close your php tags.

but I do not consider a PHP file without a closing PHP tag to be invalid and have never run into problems related to not using closing PHP tags in PHPT tests (that I am aware of).